libs/platform/auth/login/login.component.ts

```diff
@@ -28,8 +29,6 @@ export class AuthLoginComponent extends ContentMixin<LoginOptions>(LitElement) {
   @property() passwordName = 'password';
   @property() rememberMeName = 'rememberme';
```

Comment:
Do we need `emailName`, `passwordName`, and `rememberMeName`? If not, let's drop them.